python/python/tests/test_s3_ddb.py

```diff
@@ -24,11 +24,11 @@
 # These are all keys that are accepted by storage_options
 CONFIG = {
     "allow_http": "true",
-    "aws_access_key_id": "ACCESSKEY",
-    "aws_secret_access_key": "SECRETKEY",
-    "aws_endpoint": "http://localhost:9000",
-    "dynamodb_endpoint": "http://localhost:8000",
-    "aws_region": "us-west-2",
+    "aws_access_key_id": "ACCESS_KEY",
+    "aws_secret_access_key": "SECRET_KEY",
```

Comment:
Didn't you specify these as `ACCESSKEY` and `SECRETKEY` in the docker compose?

Reply:
Oh good catch. It turns out, localstack doesn't check these https://docs.localstack.cloud/references/credentials/#:~:text=The%20value%20of%20the%20secret%20access%20key%20are%20currently%20ignored%20by%20LocalStack.